code/modules/projectiles/ammunition.dm

```diff
@@ -311,34 +311,36 @@
 
 /obj/item/ammo_magazine/resolve_attackby(atom/A, mob/user)
 	//Clicking on tile with no collectible items will empty it, if it has the verb to do that.
-	if(isturf(A))
-		src.quick_empty(A)
+	if(isturf(A) && !A.density)
+		dump_it(A)
 		return TRUE
 	return ..()
 
-/obj/item/ammo_magazine/verb/quick_empty(var/turf/target)
+/obj/item/ammo_magazine/verb/quick_empty()
 	set name = "Empty Ammo Container"
 	set category = "Object"
+	set src in view(1)
 
 	if((!ishuman(usr) && (src.loc != usr)) || usr.stat || usr.restrained())
```

Comment:
```suggestion
	if(!ishuman(usr) || usr.incapacitated())
```
`incapacitated()` - usefull proc. Check for stat, restrained, paralize, sleeping, etc.
Defination of it located in [mob.dm](https://github.com/discordia-space/CEV-Eris/blob/master/code/modules/mob/mob.dm#L169)